### DISCUSSION TITLE
Hypervisor core option

### DISCUSSION THREAD

**Discussion by Torinde** [86Box/86Box#3670] [Ideas]
**Is your feature request related to a problem? Please describe.**
Faster CPU, potentially also enabling more instructions/features

**Describe the solution you'd like**
To add an even faster hypervisor core, for example based on [virt86](https://github.com/StrikerX3/virt86) as discussed at https://github.com/joncampbell123/dosbox-x/issues/1089

**Describe alternatives you've considered**
VirtualBox, VMware, QEMU

**Additional context**
WinXP will greatly [benefit](https://github.com/86Box/86Box/discussions/2909), especially later games. Although dual/quad-cores may be needed for some of those (can 86box be expanded to emulate multiple cores?)
There is [patcher9x](https://github.com/JHRobotics/patcher9x) which allows all Win9x versions to run on Ryzen and other modern CPUs.
Hypervisor GPU is discussed at https://github.com/JHRobotics/softgpu/issues/13#issuecomment-1675636496

**Comment by cold-brewed** [comment]
Virtualization is not in scope for 86Box. See [The FAQ](https://86box.readthedocs.io/en/latest/usage/faq.html#what-is-the-difference-between-86box-and-applications-like-virtualbox-or-virtual-pc) for more information.

**Comment by Torinde** [comment]
I interpreted @OBattler comments [here](https://github.com/JHRobotics/softgpu/issues/13#issuecomment-1675299897) as being receptive of hypervisor GPU and CPU.

**Comment by OBattler** [comment]
GPU passthrough only, not hyeprvisor CPU.

**Comment by kjliew** [comment]
Hmm ... Typical. Do not despair @Torinde. These "two paragons of virtue" are not the only devs around here. Behold for one's fear for being damned to speak so DARK a prophecy... 🤣

The Beginners naively interpret every instructions.
The Enlightens bring forth dynamic re-compilers.
The Wise Virtualize.
The FOOLS' pretense "Not in the scope".

**Comment by OBattler** [comment]
- kjliew: I told you why, the kind of stuff that would benefit from virtualization is beyond this emulator's scope in the first place. I also told you why we are limiting the scope - because we are understaffed and have enough work already on our plate. If you want to undertake the effort of adding a virtualizer core to the emulator and all the rework that would come with it - we currently synchronzie every single device's timings to the emulated CPU, that would have to be reworked completely to run in a parallel tread, synchronized to host time, and every essential infrasturcture (DMA, PIC, etc.) be written to eb thread-safe -, be my guest.
